src/components/utilities/Button/Button.js

```diff
@@ -0,0 +1,17 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => (
+  <Fragment>
+    <button type={props.type}
+      className={`button ${props.classes}`}>{props.children}</button>
+  </Fragment>
```

Comment:
Nice work however please update your use of the React fragment Tag

Reply:
@gbols I agree with what you are saying, it would have been better if I wrote it like this

```
<button type={props.type}
>
{props.children}
</button>
```
But the reason I am using fragment is becasue I am using it in my test.

Reply:
@gbols This has been resolved